board/views.py

```diff
@@ -35,6 +37,30 @@ class GetPosts(View):
     requests for more post information and the server responds correspondingly.
     """
 
+    def get_post_dict(post):
+        """
```

Comment:
I'm assuming that I missed the docstrings like this but we can add an issue called "fix inconsistencies and docstrings"

Reply:
Yes

Reply:
can you add the issue 😉